examples/pnpm-lock.yaml

```diff
@@ -17,13 +18,14 @@ specifiers:
 
 dependencies:
   '@angular/compiler': 14.2.8_@angular+core@14.2.8
-  '@angular/compiler-cli': 14.2.8_4i3i6fp4ncuzdz7mihas2rstji
+  '@angular/compiler-cli': 14.2.8_e2368f15fc68a991e7ec41c12d46534a
```

Comment:
Why is angular being updated?

Reply:
probably because I used an old version of pnpm to generate the lock file.